src/Nat.mo

```diff
@@ -1,72 +1,299 @@
-/// Natural numbers
+/// Utility functions for working with natural numbers.
 ///
 /// Most operations on natural numbers (e.g. addition) are available as built-in operators (e.g. `1 + 1`).
 /// This module provides equivalent functions and `Text` conversion.
+///
+/// Import from the base library to use this module.
+/// ```motoko name=import
+/// import Nat "mo:base/Nat";
+/// ```
 
 import Int "Int";
 import Order "Order";
 import Prim "mo:⛔";
+import Char "Char";
 
 module {
 
   /// Infinite precision natural numbers.
   public type Nat = Prim.Types.Nat;
 
-  /// Conversion.
-  public let toText : Nat -> Text = Int.toText;
+  /// Converts a natural number to its textual representation.
```

Comment:
Detail (unsure whether worth mentioning): Without underscore formatting for thousand digit separators.

Reply:
Will add